src/components/templates/Camp/components/MapRoomOverlay.tsx

```diff
@@ -139,6 +139,8 @@ export const MapRoomOverlay: React.FC<MapRoomOverlayProps> = ({
     (attendances[`${venue.name}/${room.title}`] ?? 0) +
     (room.attendanceBoost ?? 0);
 
+  if (!room.isEnabled) return <></>;
```

Comment:
Fragments are only really meant for grouping other sibling JSX elements together when we don't want to emit a `<div>`/similar to give them a common parent. In this case, we would want to return `null`

- https://reactjs.org/docs/conditional-rendering.html#preventing-component-from-rendering